apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -269,4 +270,20 @@ public static Character parseCharacter(String charText) {
       return Optional.of(parseCharacter(charText));
     }
   }
+
+  /**
+   * @param inputString {@link String} that possible contains invalid characters
+   * @return an {@link String} with a null representation stripped from it if the input has a null
+   *     field representation.
+   */
+  public static String removeInvalidCharacters(String inputString) {
+    String verifiedInputString = inputString;
+
+    if (!Strings.isNullOrEmpty(verifiedInputString)) {
+      // remove all NULLs
+      verifiedInputString = verifiedInputString.replaceAll("\\x00", "");
```

Comment:
This really didn't look right to me, based on my reading of [Pattern](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html) (the "Backslashes, escapes, and quoting" section, in particular). However, I was wrong!

Based on some local testing, both of these code samples work (i.e. print out "fixed"):

This:
```java
public static void main(String[] args) {
  String valueToFix = "Howdy!\u0000";
  String valueFixed = valueToFix.replaceAll("\u0000", "");
  if (valueToFix.contentEquals(valueFixed))
    System.out.println("not fixed");
  else
    System.out.println("fixed");
}
```

And this:
```java
public static void main(String[] args) {
  String valueToFix = "Howdy!\u0000";
  String valueFixed = valueToFix.replaceAll("\\x00", "");
  if (valueToFix.contentEquals(valueFixed))
    System.out.println("not fixed");
  else
    System.out.println("fixed");
}
```

I still don't understand _why_ the second one works, but it does. So: neat!